doc/dev_guide/internals/replication/orphan.rst

```diff
@@ -93,9 +87,9 @@ It is being called again in order to perform recovery.
     1.  Perform :ref:`recovery <internals-recovery_process>` from the last local
         snapshot and the WAL files.
 
-    2.  Connect to at least
-        :ref:`replication_connect_quorum <cfg_replication-replication_connect_quorum>`
-        nodes. If failed -- set status to 'orphan'.
+    2.  Connect to at least a half or more of other nodes
+        (see :ref:`bootstrap_strategy <cfg_replication-bootstrap_strategy>`).
```

Comment:
Should be:

Try to establish connections to all other nodes for `replication_connect_timeout`.
Once `replication_connect_timeout` passes or all the connections are established, proceed to "sync" stage with all the gathered connections (doesn't matter how many there are: even 1, or 0, or all).